statsmodels/tsa/statespace/mlemodel.py

```diff
@@ -2894,6 +2894,76 @@ def zvalues(self):
         """
         return self.params / self.bse
 
+    def get_rsquared(self, baseline="rwdrift", **kwargs):
```

Comment:
I'd replace `kwargs` with just a keyword argument `seasonal` that defaults to `None`.

Reply:
maybe MLEResults users want to specify parameters for different type of baseline models, like 'order1or2 difference'? so still leave kwargs there. it looks a bit neat as well